src/Ide/Plugin/Floskell.hs

```diff
@@ -1,5 +1,4 @@
 {-# LANGUAGE RecordWildCards #-}
-{-# LANGUAGE CPP #-}
```

Comment:
change not relevant, should I revert this? It is not needed, though

Reply:
keep it, cleaning up code is a good thing to do, all the time